.github/workflows/kind-e2e.yaml

```diff
@@ -18,9 +18,9 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-        - v1.19.11
-        - v1.20.7
-        - v1.21.1
+        - v1.22.9
+        - v1.23.6
+        - v1.24.0
```

Comment:
lol,

It looks like there's no overlap in k8s versions between before and now? That's exciting.